ios/utils/NSArray+RNSUtil.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSArray (RNSUtil)
+
+- (BOOL)rns_isEmpty;
+- (BOOL)rns_isNotEmpty;
+
+@end
+
+NS_ASSUME_NONNULL_END
```

Comment:
We should have done it long time ago...

TODO: replace all conditions of shape `someArray.count > 0` with call to one of these methods.

I used naming convention presented in Objective-C [docs](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html#//apple_ref/doc/uid/TP40011210-CH6-SW1) to avoid name clashing and potential undefined behaviour.

Reply:
Why would we need this? count > 0 seems straightforward enough.

Reply:
Tbh I'm baffled that `Objective-C` does not have this like every other civilised language (I'm thinking of `Kotlin`, `Scala`, `Swift`, `Rust`, even `C++` & `Java`)

It reads better, expresses intention in more explicit manner, less error prone. Only con I see here is a little weird spelling due to the prefix. 

I mean I can see it not being in scope of this PR, so if you insist on not having it right now I'll revert. Let me know.

Reply:
I see the point of it, but I would choose better names for those methods - maybe something like `__isEmpty` if we mean to use it internally? `rns` doesn't look good here though.

Reply:
I see the point of it, but I would choose better names for those methods - maybe something like `__isEmpty` if we mean to use it internally? `rns` doesn't look good here though.

Reply:
Yeah, let's discuss if we need this later and then add it in another PR if needed.

Reply:
https://github.com/software-mansion/react-native-screens/pull/1852/commits/6e97b33e0814136c924727a8b99e37335dac3d69